http_server.ts

```diff
@@ -1,17 +1,35 @@
 import { RemoteInfo } from "dgram";
-import { readFileSync } from "node:fs";
+import { readFileSync, existsSync } from "node:fs";
 import http from "node:http";
 
 import { opt } from "./options.js";
 import { discoverDevices } from "./discovery.js";
-import { DevSerial } from "./impl.js";
+import { DevSerial, SendDevStatus } from "./impl.js";
 import { Handlers, makeSession, Session, startVideoStream } from "./session.js";
 
-let BOUNDARY = "a very good boundary line";
-let responses: Record<string, http.ServerResponse[]> = {};
-let audioResponses: Record<string, http.ServerResponse[]> = {};
-let sessions: Record<string, Session> = {};
+const BOUNDARY = "a very good boundary line";
+const responses: Record<string, http.ServerResponse[]> = {};
+const audioResponses: Record<string, http.ServerResponse[]> = {};
+const sessions: Record<string, Session> = {};
 
+// Text file containing the mapping of camera names.
+const nameFile = "cameras.txt";
+
+// Reads the simple mapping of camera names from the text file.
+const cameraNames = Object.assign({},
```

Comment:
I think this would be better as a JSON file - then it'd also be fairly trivial to rename the cameras through the UI

Reply:
Personally I think that a JSON file too cumbersome for manual editing (too many curly braces and no comments at all), specially in a simple case like this (a simple ID/description pair). Yes, I agree that it would be appropriate for storing configs handled by the UI (specially if we had more settings), but I honestly doubt such feature will be added to the server anytime soon.

If you feel strongly inclined to using JSON I'll go ahead and change it, as I don't want to maintain a fork of your code and miss any enhancements to the protocol handling and server (my setup is a set-and-forget one, so I'll have to deal with it only once).